src/components/dashboard/ClaimButton.js

```diff
@@ -214,8 +215,8 @@ const getStylesFromProps = ({ theme }) => ({
     borderStyle: 'solid',
     height: isSmallDevice ? 140 : getDesignRelativeHeight(196),
     width: isSmallDevice ? 140 : getDesignRelativeHeight(196),
-    boxShadow: '10px 12px 25px -14px',
     alignItems: 'center',
+    ...getShadowStyles('10px 12px 25px -14px'),
```

Comment:
ok, but what about the shadow for native?
dont fix it as we have a new PR with new design

Reply:
with new design we can add shadow for both iOS & android, it will need design approval as values might differ a bit